tests/ui/components/results/paginationcomponent.js

```diff
@@ -17,6 +17,7 @@ const createCore = () => {
   const persistentStorage = {};
   const storage = new Storage().init();
   storage.set(StorageKeys.VERTICAL_RESULTS, { searchState: SearchStates.SEARCH_COMPLETE, resultsCount: 21 });
+  storage.set(StorageKeys.NO_RESULTS_CONFIG, { displayAllResults: true });
```

Comment:
are we going to remove this in the globalStorage call above at a later time (e.g. when we do SEARCH_OFFSET AND SEARCH_CONFIG?)

Reply:
nope we can remove this right now!